zds/search/utils.py

```diff
@@ -273,10 +270,14 @@ def delete_document(self, document):
         doc_type = document.get_search_document_type()
         doc_id = document.search_engine_id
 
-        # This can happen in tests or, for instance, we move a not-yet-indexed topic to a private forum.
-        # Try/except/pass while https://github.com/typesense/typesense-python/issues/65 is not fixed
-        with contextlib.suppress(TypesenseObjectNotFound):
-            answer = self.engine.collections[doc_type].documents[doc_id].delete()
+        # Content can be not found in tests or, for instance, we move a
```

Comment:
Je trouve difficile de comprendre "Content can be not found in tests" en première lecture. Peut-être ajouter des guillemets autour de "not found" ou utiliser un synonyme (unfound, missing...) ?